### PR TITLE
Fix max length warnigns

### PR DESCRIPTION
**Pull request by xitronix**

### CODE REVIEW

File: universal-login-contracts/lib/encode.ts [hunk 1]
```diff
@@ -42,6 +42,7 @@ export function setupInitializeWithENSArgs({keyPair, ensDomainData, gasPrice, ga
   const ensName = `${name}.${domain}`;
   const hashLabel = utils.keccak256(utils.toUtf8Bytes(name));
   const node = utils.namehash(ensName);
-  const args = [keyPair.publicKey, hashLabel, ensName, node, ensDomainData.ensAddress, ensDomainData.registrarAddress, ensDomainData.resolverAddress, gasPrice, gasToken];
+  const args =
```
Comment (JustynaBroniszewska) [inline universal-login-contracts/lib/encode.ts]: Inline

Comment (xitronix) [reply universal-login-contracts/lib/encode.ts]: changed for: 
```ts
  const args = [
    keyPair.publicKey,
    hashLabel,
    ensName,
    node,
    ensDomainData.ensAddress,
    ensDomainData.registrarAddress,
    ensDomainData.resolverAddress,
    gasPrice,
    gasToken,
  ];
```

**Comment by rzadp** [comment]
> Why not just remove this rule? It would be better and easier.

Agree - let's remove the rule

**Comment by xitronix** [comment]
If we have legth of line > 160, it often means that code is unreadable.

**Comment by xitronix** [comment]
So

**Pull request closed without merging**